scheduling_resources/scheduling-logging-resources.adoc

```diff
@@ -0,0 +1,44 @@
+include::_attributes/common-attributes.adoc[]
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-06-24
+:_mod-docs-content-type: ASSEMBLY
+
+[id="scheduling-logging-resources"]
+= Scheduling logging resources
+
+toc::[]
+
+:context: scheduling-logging-resources
+
+You can control logging resources scheduling by defining node selectors, taints and tolerations, and affinity and anti-affinity configurations. 
+
+Node selector:: A node selector specifies a map of key/value pairs that are defined using custom labels on nodes and selectors specified in pods. For the pod to be eligible to run on a node, the pod must have the same key/value node selector as the label on the node.
+
+Taints and toleration:: Taints and tolerations allow the node to control which pods should, or should not, be scheduled on them.
+
+Affinity and anti-affinity:: Pod affinity and pod anti-affinity allow you to constrain which nodes your pod is eligible to be scheduled on based on the key-value labels on other pods.
+
+[IMPORTANT]
+====
+If you are using node selectors and node affinity in the same pod configuration, the following rules control pod placement onto nodes:
+
+* If you configure both `nodeSelector` and `nodeAffinity`, both conditions must be satisfied for the pod to be scheduled onto a candidate node.
+
+* If you specify multiple `nodeSelectorTerms` associated with `nodeAffinity` types, then the pod can be scheduled onto a node if one of the `nodeSelectorTerms` is satisfied.
+
+* If you specify multiple `matchExpressions` associated with `nodeSelectorTerms`, then the pod can be scheduled onto a node only if all `matchExpressions` are satisfied.
```

Comment:
```suggestion
* If you specify multiple `matchExpressions` types associated with `nodeSelectorTerms`, then the pod can be scheduled onto a node only if all `matchExpressions` are satisfied.
```

Reply:
I don't know why we are mentioning `nodeSelectorTerms` because the example does not use them, nor does the module define what they are so the user has no way of knowing how to use these or what they are.